integration_tests/dbt_project.yml

```diff
@@ -24,9 +24,18 @@ dispatch:
 seeds:
   +quote_columns: false
 
+# FIXME: query-comment must be disabled for Athena to work because /* block comments are unsupported in Athena DML
+# Removing this line will result in a Runtime Error during the integration test 
+#   `2 of 5 (2) create external table dbt.people_csv_partitioned ...`. The error is
+#   "FAILED: ParseException line 1:0 cannot recognize input near '/' '*' '{".
+# Is there a better way around this?
+query-comment:
```

Comment:
Yes, this is something the adapter can reimplement! That's where the logic lives, with just this sort of reason in mind. The base implementation is defined on the core adapter interface: [`_QueryComment.add()`](https://github.com/dbt-labs/dbt-core/blob/10b27b963392cce33950d9cef31c7da5c3e65592/core/dbt/adapters/base/query_headers.py#L32-L45) in `core.dbt.adapters.base.query_headers`

The right place for this is probably a new issue in the `dbt-athena` adapter

Reply:
nice, thanks for the pointer! I've opened up a PR in the dbt-athena adapter repo.

Reply:
nvm. found it: https://github.com/Tomme/dbt-athena/pull/64

Reply:
What do you think of it?